js-intro.js

```diff
@@ -4,14 +4,26 @@
 // Please read all questions thoroughly
 // Psuedo coding is highly recommended
 
-// --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
-// Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+// --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit. Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+boilingPointCheck = (temp) => {
+  if (temp < 212) {
+    return `${temp} is below boiling point`
+  } else if (temp > 212) {
+    return `${temp} is above boiling point`
+  } else if (temp = 212) {
```

Comment:
Thx for using `else if` rather than `else` to display your intention with your code